openslides/global_settings.py

```diff
@@ -11,7 +11,7 @@
 
 AUTHENTICATION_BACKENDS = ('openslides.users.auth.CustomizedModelBackend',)
 
-LOGIN_URL = '/login/'
+LOGIN_URL = '/users/'
```

Comment:
Ah I understand. Then we should remove the redirection feature in Django and this settings value. Or redirect to  '/' ans let AngularJS do the rest (checking session and displaying the login form if user is not authenticated).